android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -779,10 +746,7 @@ private void initShowOnLockScreenPrefsCallbacks()
 
   private void removePreference(@NonNull String categoryKey, @NonNull Preference preference)
   {
-    PreferenceCategory category = findPreference(categoryKey);
-    // TODO: check whether it's needed #2049
-    if (category == null)
-      return;
+    PreferenceCategory category = getPreference(categoryKey);
```

Comment:
```suggestion
    final PreferenceCategory category = getPreference(categoryKey);
```

Reply:
Actually, in this and other similar cases, we can avoid creating the category variable, and make the code a bit shorter/cleaner, right?

Reply:
`getPreferences` method is generic method. And variable type `PreferenceCategory` tells actual return type. I'll think on getting rid of local var.